### DISCUSSION TITLE
Soundswitch can't find headset after windows update

### DISCUSSION THREAD

**Discussion by Crabmeat1432** [Belphemur/SoundSwitch#1358] [Q&A]
Hello,  I did an automatic update to the latest Windows 11 last night and now Soundswitch can't find my headset.  I tried updating SS to 6.8.1.0 but that didn't help.

![image](https://github.com/Belphemur/SoundSwitch/assets/123743766/3244e59c-663b-40e1-a984-7d23a034b736)
![image](https://github.com/Belphemur/SoundSwitch/assets/123743766/4aefff38-91ee-4528-bda5-0aafd8e2db53)
![image](https://github.com/Belphemur/SoundSwitch/assets/123743766/67e384ce-091e-4294-b0c6-97f84b76ef54)

MIght anyone know how to get the headset connected to Soundswitch again?  Thank you.

**Comment by Crabmeat1432** [comment]
Update - I should mention that this only happens when using the profile hot keys.    When I select the headphone from the Notification Tray it switches properly.

**Comment by estradamus** [comment]
The issue starts for me after upgrading to Windows 11. I found out that when the Bluetooth device is off, win11 disables the device and hides it from the "playback" and "recoding" lists under the "sound" setting, but Windows 10 always keeps the Bluetooth devices enabled.

Soundswitch repeatedly shows the device isn't available until it removes the unavailable Bluetooth device from the saved hotkey configuration.

I have two options:

1. I must turn on my Bluetooth headset before clicking the hotkeys.
2. If I click the hotkeys before turning on my Bluetooth headset, I have to go to Soundswitch hotkey profiles and update my Bluetooth headset profile again.
![Airpods1](https://github.com/Belphemur/SoundSwitch/assets/44001423/9f856b67-598c-4452-af81-a70cc43c273d)
![Airpods2](https://github.com/Belphemur/SoundSwitch/assets/44001423/0cb4115d-8d70-43ef-9b69-124f4dde0896)
